cypress/e2e/admin/administrateMigration.feature

```diff
@@ -0,0 +1,26 @@
+@release
+Feature: Admin Users - To administrate the Migration process.
+
+  As an Admin I want to start, finish and make the migration mandatory.
+
+  Background:
+    Given I am logged in as a 'admin' at 'nbc'
+    When I go to administration page
+    When I go to school administration
+
+  Scenario: Admin starts the migration and the School Number is added
+    When I can see the school number
+    Then I can click on the start migration button
+    Then I can see the migration information text
+    Then I can write an email to a pre-defined receiver
+    Then I can click on the information link that opens a new tab
```

Comment:
This step is not clear if it is an action or an assertion.
Actions (When) should be with the concrete step ("When I click ...") and assertions (Then) with what you check ("Then I see the correct information link url" or similar).
The method just checks the href url and not what happens, if you click on it.

You could also use parameters to write the correct url here:
"Then I see the information link href is 'https://blog.niedersachsen.cloud/umzug' "